client/lib/upgrades/actions/checkout.js

```diff
@@ -29,6 +35,21 @@ function setNewCreditCardDetails( options ) {
 	} );
 }
 
+function submitFreeTransaction( site, cartItem, onComplete ) {
```

Comment:
I think it would be moe consistent to give a `cart` instead of a `cartItem` here